config/forcedPhotCoadd.py

```diff
@@ -0,0 +1,44 @@
+import os.path
+
+from lsst.meas.base import CircularApertureFluxAlgorithm
+
+config.measurement.load(os.path.join(os.path.dirname(__file__), "apertures.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "kron.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "convolvedFluxes.py"))
+config.measurement.load(os.path.join(os.path.dirname(__file__), "gaap.py"))
+config.load(os.path.join(os.path.dirname(__file__), "cmodel.py"))
+
+config.catalogCalculation.plugins.names = ["base_ClassificationExtendedness"]
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)